### PR TITLE
Starting codeowners files.

### PR DESCRIPTION
**Pull request by AMZN-scspaldi**
Signed-off-by: scspaldi <scspaldi@amazon.com>

### CODE REVIEW

**Comment by AMZN-scspaldi** [comment]
This is a first pass at sig-testing ownership.

**Review by Kadino — COMMENTED** [review]
To add:
/o3de/scripts/ctest/
/o3de/pytest.ini
/o3de/AutomatedTesting/Gem/PythonTests/editor_test_testing/

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
```
Comment (evanchia-ly-sdets) [inline .github/CODEOWNERS:1]: Do we need any license or description headers?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+/cmake/LYTestWrappers.cmake @o3de/sig-testing-reviewers
+
```
Comment (evanchia-ly-sdets) [inline .github/CODEOWNERS:3]: nit- is there a reason for this whitespace?

**Review by evanchia-ly-sdets — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (AMZN-scspaldi) [reply .github/CODEOWNERS:1]: Other git configuration files don't have them so I wasn't going to include them.

**Review by evanchia-ly-sdets — APPROVED** [review]

**Review by Kadino — APPROVED** [review]
Interesting that it's sig-testing-reviewers due to repo permissions. This may land awkwardly for certain SIGs, but that permissions management is currently out of scope.

**Pull request merged**